tests/unit/mainwindow/statusbar/test_percentage.py

```diff
@@ -29,6 +29,8 @@
 def percentage(qtbot):
     """Fixture providing a Percentage widget."""
     widget = Percentage()
+    # Force immedate update of percentage widget
+    widget.set_perc.throttle_set(-1)
```

Comment:
I'm not sure how to fix the lint and mypy warnings on this line. I could just ignore them, but I feel like the linter should know this exists.

Reply:
Given that each instance of the `throttle` class is only ever going to be called on one function object, it may make sense to set some property in `__call__` and then return `self`.

Reply:
Looking at [python implementation of `lru_cache`](https://github.com/python/cpython/blob/master/Lib/functools.py#L636), they assign cache_clear dynamically too, so the linter probably just assume it exists.

We can simply [suppress the message](https://stackoverflow.com/questions/35990313/avoid-pylint-warning-e1101-instance-of-has-no-member-for-class-with-dyn)...